resolwe/flow/templatetags/process_fields.py

```diff
@@ -6,10 +6,31 @@
 """
 from django import template
 
+from resolwe.flow.models import Data
+
 
 register = template.Library()  # pylint: disable=invalid-name
 
 
+def get_data_attr(data, attr):
+    if isinstance(data, dict):
+        # `Data` object's id is hydrated as `__id` in expression engine
+        data = data['__id']
+
+    data_obj = Data.objects.filter(id=data)
+
+    if not data_obj.exists():
```

Comment:
We call `exists` and `first`. Are we doing 2 queries to DB or does Django handle this efficiently? Wouldn't it be more efficient to just call `first` with a try/catch?